1.chunkedgraph.md

```diff
@@ -92,10 +98,17 @@ $ kubectl exec -ti deploy/master -- bash
 > ingest status
 2       : 8 / 64 # progress at each layer
 3       : 1 / 8
-4       : 0 / 1
+4       : 0 / 1 # this graph has 4 layers
 ```
 Output should look like this if successful. Now you can rerun the ingest without the `--test` flag (make sure to use a different bigtable name).
 
+> NOTE: The number of `workerDeployments` in `values.yaml` relates to the number of layers in the graph. For example, a graph with 6 layers must have 5 sections in `workerDeployments` (l2 - l6). To remove depoyments for a specific layer, uncomment the related section in `workerDeployments`. Also, if you add or remove worker deployments, `trackerDepoyments` need to be adapted accordingly. Note that the highest layer does not need a tracker because it has only one final root chunk. 
```

Comment:
Thank you for extending the docs! Just one thing, instead of commenting and uncommenting which can feel cumbersome, you could simply set the `enabled` key to `true` / `false` as needed.

I will merge this for now and update the docs to reflect this.